### PR TITLE
Add show for home page places

### LINKED ISSUE #76: User side: available place show page

### PR DESCRIPTION
**Pull request by ChernikovVolodymyr**
Add show for home page places
Add fa-awesome
Fix logout for user

image

image

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -8,4 +9,14 @@ def index
...
   end
+
+  def show
+    @place
+  end
+
+  private
+
+  def set_place
+    @place = Place.find(params[:id])
```
Comment (oleksandr-kychun) [inline app/controllers/home_controller.rb]: need to check whether provided place id belongs to available places for given user

File: app/views/home/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+</br>
+<div class="button fa fa-arrow-left fa-lg pull-right">
+  <%= link_to 'Back', root_path, class: "btn btn-outline-dark" %>
+</div>
+
+<h2>Place <%= @place.id %></h2>
+
+  <ul>
+    <li><u><b>Name:</b></u> ........................................ <%= @place.name %></li>
```
Comment (ghost) [inline app/views/home/show.html.erb]: If you want you can add `list-group-flush` class. https://getbootstrap.com/docs/5.1/components/list-group/#flush

**Review by ghost — APPROVED** [review]

File: app/views/home/show.html.erb (hunk 2 above)
Comment (ChernikovVolodymyr) [reply app/views/home/show.html.erb]: Added here https://github.com/vova/war_shelter/pull/79/commits/7d0d1c05e2236d5b1982e5cee775fef00884219b

Comment (ghost) [reply app/views/home/show.html.erb]: Thanks!

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (ChernikovVolodymyr) [reply app/controllers/home_controller.rb]: All places have specific id,  and show endpoint passes this id.

Comment (oleksandr-kychun) [reply app/controllers/home_controller.rb]: when you pass in random id, access should be denied
we need to check, that passed id is available for certain user

Comment (ChernikovVolodymyr) [reply app/controllers/home_controller.rb]: Added checking specific available place for current user 
Addressed here https://github.com/vova/war_shelter/pull/79/commits/559ff1a4c6a8f85021e0b4a2c720102ee30949eb

**Review by oleksandr-kychun — APPROVED** [review]

**Pull request merged**